Tests/Route/DefaultRouteGeneratorTest.php

```diff
@@ -51,6 +51,10 @@ public function testGenerateUrl($expected, $name, array $parameters)
         $collection->add('foo');
         $collection->addCollection($childCollection);
 
+        $nochildCollection = new RouteCollection('base.Code.Child', 'admin_child', '/foo/', 'BundleName:ControllerName');
```

Comment:
Try it sometimes, maybe you'll see the light ^^
You might also be tempted with emacs, or even … [spacemacs](http://spacemacs.org/)